starter-code/css/main.css

```diff
@@ -1,8 +1,112 @@
 /*
 Colors:
-
 Text: 1A1A1A
 Green: #00B172
 White: #FFF
-
 */
+
+*{
+	border: 1px solid transparent;	
+	box-sizing: border-box;
+	font-family: 'Lato', sans-serif;
+}
+
+ul{
+	float: right;
+	padding-right: 50px;
+}
+
+li{
+	list-style: none;
+	display: inline;
+    margin-left: 10px;
+}
+
+nav{
+	height: 100px;
+	margin-bottom: 40px;
+/*	position: static;
```

Comment:
You want `positon: fixed;` here. Then, you need to position it using `top: 0`.

The problem then will be that the picture below will go underneath the navbar. Indeed, because `nav` has now it fixed position, it is not affecting the normal flow of the page. The next child, which is the `div` with the picture, will then be anchored to the top of the document. We need to move the rest of the page down to an amount equal to the height of the `nav` (here `100px`) by using `margin-top` on it.

Lastly, to make sure that the `nav` is always visible, we need to increase its `z-index`. It is zero by default. To set it to 1, we can add `z-index: 1;`.